Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -137,6 +137,11 @@ function createStackEntry(props: any): any {
  *  </View>
  * ```
  *
+ * ## Navigator is deprecated and has been removed from this package. 
```

Comment:
at this point, i'm wondering if it's better to just update the example to be navigator agnostic.

@hramos